tgui/packages/tgui/interfaces/MODsuit.js

```diff
@@ -229,18 +239,20 @@ const StatusReadout = (props, context) => {
   } = props;
   return (
     <>
-      <Stack textAlign="center">
-        <Stack.Item grow>
-          <Section title="Operation Time">
-            {active ? statustime : '00:00:00'}
-          </Section>
-        </Stack.Item>
-        <Stack.Item grow>
-          <Section title="Operation Number">
-            {active ? statusid || '0' : '???'}
-          </Section>
-        </Stack.Item>
-      </Stack>
+      {!!show_time && (
+        <Stack textAlign="center">
+          <Stack.Item grow>
+            <Section title="Operation Time">
+              {active ? statustime : '00:00:00'}
+            </Section>
+          </Stack.Item>
+          <Stack.Item grow>
+            <Section title="Operation Number">
+              {active ? statusid || '0' : '???'}
```

Comment:
minor nit but this is more complex than it needs to be, I would prefer statusid to be given the default of 0 in the destructuring statement.